packages/desktop-client/src/components/reports/graphs/StackedBarGraph.tsx

```diff
@@ -116,26 +117,15 @@ type StackedBarGraphProps = {
   data;
   balanceTypeOp;
   compact: boolean;
-  domain?: {
-    y?: [number, number];
-  };
 };
 
-function StackedBarGraph({
-  style,
-  data,
-  balanceTypeOp,
-  compact,
-  domain,
-}: StackedBarGraphProps) {
+function StackedBarGraph({ style, data, compact }: StackedBarGraphProps) {
+  let privacyMode = usePrivacyMode();
   const colorScale = getColorScale('qualitative');
 
-  const getVal = (obj, key) => {
-    if (balanceTypeOp === 'totalDebts') {
-      return -1 * obj[key].amount;
-    } else {
-      return obj[key].amount;
-    }
+  const CustomTick = (value: string, index: number) => {
+    if (!privacyMode) return value;
+    return '...';
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think I saw this same component used in a bunch of places. Would make sense to extract it and re-use.